lms/static/scripts/frontend_apps/api-types.ts

```diff
@@ -147,8 +147,13 @@ export type YouTubeVideoInfo = {
  * Dashboard-related API types
  */
 
+/**
+ * Represents dates exposed by the backend as string, in ISO format
+ */
+export type ISODate = string;
```

Comment:
```suggestion
export type ISODateTime = string;
```

The name makes it sound like it is just the date, with no timestamp component. We should always make it clear whether a value is just a date, or both a date and time.